kqueen/models.py

```diff
@@ -30,6 +31,8 @@ class Cluster(Model, metaclass=ModelMeta):
     state = StringField()
     kubeconfig = JSONField()
     metadata = JSONField()
+    created_at = DatetimeField()
+    deleted_at = DatetimeField()
```

Comment:
Do you plan set `deleted_at` instead of deleting objects? It won't make sense IMO.

Reply:
I will remove deleted_at field